config/chief.php

```diff
@@ -118,4 +118,9 @@
      * 'quill' and 'redactor'. By default the free quill editor is set.
      */
     'editor' => 'quill',
+
+    'healthMonitor' => [
```

Comment:
zou hier nog comment toevoegen a la:
Here you can define the stack of checks for the health monitoring in Chief.